cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -906,7 +912,9 @@ subroutine seabed_stress_factor_LKD (nx_block, ny_block,         &
          hwu = min(hwater(i,j),hwater(i+1,j),hwater(i,j+1),hwater(i+1,j+1))
 
 !         if (hwu < threshold_hw) then
-         docalc_tbu = max(sign(c1,threshold_hw-hwu),c0)
+!         docalc_tbu = max(sign(c1,threshold_hw-hwu),c0)
```

Comment:
can we remove this commented line?

Reply:
Removed. I also squashed the last three commits that were review related